<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolButton" name="Button_Generate">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>GENERATE</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_grid" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>502</width>
      <height>502</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="Button_Clean">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>490</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>CLEAN</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_height">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_height">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>HEIGHT</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_width">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>360</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_width">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>360</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>WIDTH</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_point1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>POINT 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_point2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>230</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>POINT 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_x">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_point1_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>200</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_x_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_point1_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>250</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_x_point_1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_y_point_1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_x_point_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_y_point_2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>250</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QToolButton" name="Button_Solution">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>290</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>SOLUTION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>181</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate</family>
      <pointsize>23</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;MAZE&lt;br/&gt;GENERATOR&lt;br/&gt;AND SOLVER&lt;br/&gt;&lt;br/&gt;by &lt;span style=&quot; font-weight:700;&quot;&gt;DYAHDEME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_File"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="actionSave_maze_as_txt"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuSave"/>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionSave_maze_as_txt">
   <property name="text">
    <string>Save maze as .txt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
